manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Rainmeter.Rainmeter.Beta
+PackageVersion: 4.4.0.3506
```

Comment:
```suggestion
PackageVersion: 4.4 beta r3506
```

Reply:
I think leaving a full version on this line is better.

Reply:
This is the version you guys are displaying in the Control Panel, which is why it has to match that.